05-visualisation-ggplot2.Rmd

```diff
@@ -152,7 +152,7 @@ ggplot(surveys_complete)
 - define aestetics (`aes`), that maps variables in the data to axes on the plot
      or to plotting size, shape color, etc.,
 
-```{r}
+```{r, eval=FALSE}
```

Comment:
Initially, I think the error was intended to show that using just the function `ggplot` wasn't enough to produce a plot and that a `geom_` function was needed.